### PR TITLE
OSOE-482: Enforce parameter splatting instead of backtick in Lombiq.Analyzers.PowerShell

### PR DESCRIPTION
**Pull request by BenedekFarkas**
[OSOE-482](https://lombiq.atlassian.net/browse/OSOE-482)

### CODE REVIEW

**Review by Piedone — CHANGES_REQUESTED** [review]
I'd prefer not to do this tab stop-alignment here either.

**Comment by BenedekFarkas** [comment]
> I'd prefer not to do this tab stop-alignment here either.

Agreed, for reference, I replied on another PR: https://github.com/Lombiq/Utility-Scripts/pull/28#discussion_r1051637023

File: PowerShell/Azure/Sql/Get-AzureWebAppSqlDatabase/Get-AzureWebAppSqlDatabase.psm1 [hunk 1]
```diff
@@ -8,21 +8,20 @@
...
     Get-AzureWebAppSqlDatabase @{
-        ResourceGroupName    = "YeahSubscribe"
-        WebAppName           = "EverythingIsAnApp"
-        SlotName             = "Staging"
+        ResourceGroupName = "YeahSubscribe"
+        WebAppName = "EverythingIsAnApp"
+        SlotName = "Staging"
         ConnectionStringName = "Nokia"
-    }
+    }
```
Comment (Piedone) [inline PowerShell/Azure/Sql/Get-AzureWebAppSqlDatabase/Get-AzureWebAppSqlDatabase.psm1]:
```suggestion
    }
```

File: .github/workflows/static-code-analysis.yml [hunk 2]
```diff
@@ -7,10 +7,6 @@ on:
     branches:
       - dev
 
-defaults:
```
Comment (Piedone) [inline .github/workflows/static-code-analysis.yml:10]: See my notes about the workflow here: https://github.com/Lombiq/Utility-Scripts/pull/28#discussion_r1052640912

**Review by Piedone — CHANGES_REQUESTED** [review]

File: PowerShell/Azure/Sql/Get-AzureWebAppSqlDatabase/Get-AzureWebAppSqlDatabase.psm1 (hunk 1 above)
Comment (BenedekFarkas) [reply PowerShell/Azure/Sql/Get-AzureWebAppSqlDatabase/Get-AzureWebAppSqlDatabase.psm1]: Not sure how these can creep back in sometimes, but it's really annoying that removing trailing whitespaces is not a default step of code cleanup (Ctrl+K+D with the keymap imported from VS). VS Code can do it too with Ctrl+K+X, but you can't bind multiple commands to the same key combination without an extension.
I updated my Razer Synapse macro to run it too before Ctrl+K+D and Ctrl+R+G (usings cleanup).

**Review by Piedone — APPROVED** [review]

**Pull request merged**